src/components/section/active-section/index.tsx

```diff
@@ -0,0 +1,17 @@
+import classNames from './active-section.module.scss';
+import Active from '../../challenges/active';
+
+const ActiveSection = (props) => {
+  let activeContent;
+
+  if (props.screen == 'Challenges') {
```

Comment:
Let's use a constant to hold the value 'Challenges' in a constants file

Reply:
@sumitd94 Was this looked into?

Reply:
Yes
Ref : https://github.com/Real-Dev-Squad/website-roadmap/pull/8/commits/08fcb4a54dff1dd083ba1c2e4cc8df90ef496daf